include/SDL3/SDL_events.h

```diff
@@ -702,8 +717,11 @@ typedef struct SDL_PenMotionEvent
     SDL_WindowID windowID;  /**< The window with pen focus, if any */
     SDL_PenID which;        /**< The pen instance id */
     Uint8 padding1;
```

Comment:
I don't think we need this padding for alignment anymore, do we?

Reply:
The current implementation needs this to match between different event types, since it assigns to all of them as if it were the PenMotion event.  :(